packages/fast-components-styles-msft/package.json

```diff
@@ -36,6 +36,6 @@
     "@microsoft/fast-jss-utilities": "^1.0.0"
   },
   "peerDependencies": {
-    "chroma-js": "^1.3.0"
+    "jss-extend": "^6.2.0"
```

Comment:
check to see if this is part of the jss presets

Reply:
Removed extend